### PR TITLE
Issue/6

### PR DESCRIPTION
**Pull request by matthijs-va**

### CODE REVIEW

**Comment by Jelle-S** [comment]
Looks good to me

File: src/AbstractRoboFile.php [hunk 1]
```diff
@@ -471,70 +476,146 @@ protected function syncTask(
...
+
+                $collection->addTask(
+                    $this->taskSsh($sourceHost, $sourceAuth)
+                        ->timeout(300)
+                        ->exec($rsync)
+                );
+            }
+        }
+
+        if ($opts['data'] || $opts['files']) {
```
Comment (daften) [inline src/AbstractRoboFile.php:553]: DOn't we have to split this up so if rsync is enabled, files aren't backupped/ synced like this?

Comment (Jelle-S) [reply src/AbstractRoboFile.php:553]: See https://github.com/digipolisgent/robo-digipolis-helpers/pull/11/files#diff-210474f4a6d0661ede81be4fa731142cR481 ;-)

Comment (Jelle-S) [reply src/AbstractRoboFile.php:553]: Oh, hang on.... if $opts['data'] === TRUE...

Comment (Jelle-S) [reply src/AbstractRoboFile.php:553]: nvm, I'm confusing myself... it's still valid. The options are passed to the backup task, so when $opts['files'] === FALSE (when rsync is used) and $opts['data'] === TRUE, only the db backup will be created

**Pull request merged**